book.js

```diff
@@ -1,52 +1,63 @@
-const books = JSON.parse(localStorage.getItem('books')) || [];
+class Books {
+  constructor() {
+    this.books = JSON.parse(localStorage.getItem('books')) || [];
+  }
 
-function addBook(title, author) {
-  const id = Date.now();
-  books.push({ title, author, id });
-  localStorage.setItem('books', JSON.stringify(books));
-}
+  addBook(title, author) {
+    const book = { title, author, id: Date.now() };
+    this.books.push(book);
+    localStorage.setItem('books', JSON.stringify(this.books));
+  }
 
-function removeBook(id) {
-  const index = books.findIndex((book) => book.id === Number(id));
-  books.splice(index, 1);
-  localStorage.setItem('books', JSON.stringify(books));
-}
+  removeBook(id) {
+    const index = this.books.findIndex((book) => book.id === Number(id));
+    if (index !== -1) {
+      this.books.splice(index, 1);
+      localStorage.setItem('books', JSON.stringify(this.books));
+    }
+  }
```

Comment:
Great job team 🤝

- [ ] [Optional] Kindly ensure your project is able to delete books from the booklist. I am able to add books but I am not able to delete books.